packages/form-js-editor/test/spec/render/components/form-fields/helper/index.js

```diff
@@ -1,9 +1,29 @@
+import { FormEditorContext } from '../../../../../../src/render/context';
 import { FormContext } from '@bpmn-io/form-js-viewer';
+import showdown from 'showdown';
+
+class MarkdownRenderer {
```

Comment:
I guess we can also use the Markdown Renderer from the viewer here, I just added one so we are covered (shouldn't make a big difference, it should work anyway).

Reply:
Since showdown uses some global configs I don't think it's all that bad to have it here, so we can control the details better :)